src/main/java/com/google/impactdashboard/database_manager/data_read/DataReadManagerImpl.java

```diff
@@ -45,6 +86,27 @@ public double getAverageIAMBindingsInPastYear(String projectId) {
    */
   @Override
   public Map<Long, Integer> getMapOfDatesToIAMBindings(String projectId) {
-    throw new UnsupportedOperationException("Unimplemented");
+    HashMap<Long, Integer> datesToBindings = new HashMap<Long, Integer>();
+    AtomicReference<Long> date = new AtomicReference<Long>(Long.parseLong("1590883200000"));
+
+    if (projectId.equals("project-id-1")) {
+      Arrays.asList(1000, 1000, 1000, 2000, 2050, 2150, 2150, 2150, 2150, 
+        1150, 1150, 1150, 1150, 2000, 2000, 2500, 2500, 2300, 2300, 1000, 
+        1000, 1000, 1100, 1100, 1000, 1000, 1300, 1300, 1350, 1350)
```

Comment:
I think that these arrays might be defined in multiple places in these files, not 100% sure. If they are, could you make them a constant in a common file somewhere?

Reply:
I had duplicated this code here because it's a temporary feature that will only exist in DataReadManagerImpl in the prototype, and will be replaced very soon with real functionality. I was also reluctant to connect it with the list in DataReadManagerFake because they're not supposed to be the same lists necessarily (there's no reason they need to be the same). I could totally pull it out into its own constant if you think it's worth it! But soon this list will only exist in DataReadManagerFake as hard-coded initialized data.